adapters/intertech/intertechtest/exemplary/simple-banner-muli-format.json

```diff
@@ -0,0 +1,115 @@
+{
+  "mockBidRequest": {
+    "id": "test-missing-dimensions-request-id",
+    "device": {
+      "ip": "123.123.123.123",
+      "ua": "Mozilla/5.0 (X11; Linux x86_64)"
+    },
+    "site": {
+      "page": "http://bannercheck.com"
+    },
+    "imp": [
+      {
+        "id": "test-imp-missing-dim",
+        "tagid": "tag-missing-dim",
```

Comment:
Nitpick: I suggest simplifying the names of the ids here. Just use `request-id-1`, `imp-id-1`, `tag-id-1`. The other names can be confusing. The test file name does a good job of indicating the test purpose.